docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -68,24 +125,33 @@ kotlin {
 <tab title="Groovy" group-key="groovy">
 
 ```groovy
-kotlin {
+tasks.named('compileKotlin', org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask.class) {
     compilerOptions {
-        apiVersion = org.jetbrains.kotlin.gradle.dsl.KotlinVersion.%gradleApiVersion%
+        apiVersion.set("1.8")
     }
 }
 ```
 
 </tab>
 </tabs>
 
+## Target the JVM
+
+As explained before, you can define compiler options for your JVM/Android projects at the extension, target, and compilation unit levels.
+
+Default JVM compilation tasks are called `compileKotlin` for production code and `compileTestKotlin`
+for test code. The tasks for custom source sets are named according to their `compile<Name>Kotlin` patterns.
+
+Regarding Android compilation tasks, you can consult them by typing the `gradlew tasks --all` command in the terminal.
```

Comment:
The "consult" here might be making this sentence more complicated that it needs to be: we can probably say "You can see the list of Android compilation tasks by running ..."

Although I don't think I understand how Android-specific this command is. This is going to be all tasks, right, not just Android? Should we provide instructions for recognizing Android-specific tasks? Even if they are simply the ones with "Android" in the name :)